### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by airon90**
Please wait for every translation being done.

* [x] full_desctiption.txt
* [x] short_description.txt
* [x] changelogs/1.txt
* [x] changelogs/2.txt 
* [x] strings.xml

### CODE REVIEW

**Comment by TecCheck** [comment]
Thank you very much. I'm currnently quite busy. I'll have a look at this pr in a few days

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="lyrics_provider_over_line">Testo da</string>
+
+    <string name="lyrics_clipboard_label">Testo</string>
+
+    <!-- Saved songs view -->
+    <string name="delete">Elimina</string>
+    <string name="items_selected">%1$d selezionati</string>
+
+    <!-- Notification permissions -->
+    <string name="permission_description">FastLyrics ha bisogno dell'accesso alle notifiche del tuo dispositivo per scoprire qual è il brano in riproduzione. Puoi concedere il permesso nelle impostazioni cliccando sul bottone sottostante</string>
```
Comment (TecCheck) [inline app/src/main/res/values-it/strings.xml:32]: The apostrophe here has to be escaped: `dell'accesso`. Like this: `dell\'accesso`

**Review by TecCheck — COMMENTED** [review]
Just one thing here. Everything else looks good to me

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="lyrics_network_exception">C\'è stato un problema di rete mentre si stava caricando il testo</string>
+    <string name="lyrics_unknown_error">C\'è stato un errore sconosciuto</string>
+
+    <string name="lyrics_provider_over_line">Testo da</string>
+
+    <string name="lyrics_clipboard_label">Testo</string>
+
+    <!-- Saved songs view -->
+    <string name="delete">Elimina</string>
+    <string name="items_selected">%1$d selezionati</string>
```
Comment (TecCheck) [inline app/src/main/res/values-it/strings.xml:29]: Since 3bf0df1 this can be a plural

Comment (TecCheck) [reply app/src/main/res/values-it/strings.xml:29]: You can take a look at [this guide](https://developer.android.com/guide/topics/resources/string-resource#Plurals) to see how plurals work

**Comment by airon90** [comment]
Thank you very much, everything seems done :)

**Comment by TecCheck** [comment]
Thank you very much

**Pull request merged**